runtime/near-vm/fuzz/Cargo.toml

```diff
@@ -10,7 +10,7 @@ cargo-fuzz = true
 
 [dependencies]
 anyhow = "1"
```

Comment:
Worthwhile to review all crate dependencies and convert to `.workspace = true` everything that's applicable while here.

Reply:
Right! I’ll do that as a follow-up PR, in order to keep the diffs smaller and easier to audit should need arise (especially I should have made the `cargo fmt` outside of this PR if I had thought of it, but I guess now’s too late to do it)